iOS/FlowWalletKit/Sources/Wallet/Wallet.swift

```diff
@@ -290,4 +290,36 @@ public class Wallet: ObservableObject {
         // Combine results from both parallel operations
         return accountList.filter{ $0.keys.hasSignleFullWeightKey }
     }
+    
+    /// Fetch Account by address and chainId
+    /// - Parameters:
+    ///   - address: The address to fetch account from
+    ///   - chainID: The network to fetch accounts from
+    /// - Returns: Account
+    public func fetchAndUpdateAccount(by address: String, chainID: Flow.ChainID) async throws -> Flow.Account? {
```

Comment:
Let's make it only update the accountInfo in the account list, instead of adding it.

Reply:
If the account is the not in the existing accounts list, let's don't add it in here.